NHAPI/NHAPI/Functions.cs

```diff
@@ -252,7 +282,7 @@ private string PrepBofArgs(string bofName, string entryPoint, string packedBofAr
         {
             //locate BOF
             string fullPath = Directory.GetCurrentDirectory();
-            if(bofName.IndexOf(":\\") > -1 && File.Exists(fullPath))
+            if(bofName.IndexOf(":\\") > -1 && File.Exists(bofName))
```

Comment:
This is a fix for small bug which prevents execution of BOFs which were specified by their full name.